app/src/main/java/com/nutomic/syncthingandroid/activities/FirstStartActivity.java

```diff
@@ -389,6 +415,19 @@ private void requestLocationPermission() {
                 Constants.PermissionRequestType.LOCATION.ordinal());
     }
 
+    @TargetApi(33)
+    private void requestNotificationPermission() {
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
I think we can skip the check here. The method is only invoked if `isNotificationPermissionGranted()` returns `false`.

Reply:
Got it.